Find a functionally equivalent code:android.os.Environment.initForCurrentUser:COMMENT
Method Modifier: public      static      hidden      
Comment:/**
 * {@hide}
 */

@@ -1,7 +1,4 @@
 {
     final int userId = UserHandle.myUserId();
     sCurrentUser = new UserEnvironment(userId);
-    synchronized (sLock) {
-        sPrimaryVolume = null;
-    }
 }

